src/site/content/en/blog/vitals-tools/index.md

```diff
@@ -1,184 +1,347 @@
 
-With Google Search now including Core Web Vitals as the foundation for evaluating [page experience](https://webmasters.googleblog.com/2020/05/evaluating-page-experience.html), it's important that these metrics are as available and actionable as possible.
+{% Aside 'key-term' %}
+[**Lab data**](/lab-and-field-data-differences/#lab-data) describes how _hypothetical_ users _may_ experience your website. [**Field data**](/lab-and-field-data-differences/#field-data) describes how _real_ users _actually_ experienced your website. Field data is also known as Real User Monitoring (RUM).
+{% endAside %}
 
-<figure class="w-figure">
-  {% Img src="image/admin/V00vjrHmwzljYo04f3d3.png", alt="Summary of Chrome and Search Tools that support the Core Web Vitals metrics", width="800", height="509", class="w-screenshot" %}
-</figure>
+A workflow for improving Core Web Vitals for your website will be explored in this guide, but where that workflow begins depends on whether you're collecting your own field data. Where it ends may depend on which of Google's tools you'll find useful in diagnosing and fixing user experience problems.
 
-{% Aside 'key-term' %}
-**Lab tools** provide insight into how a _potential user_ will likely experience your website and offer reproducible results for debugging. **Field** tools provide insight into how your _real users_ are experiencing your website; this type of measurement is often called Real User Monitoring (RUM). Each [lab or field tool](/how-to-measure-speed/#lab-data-vs-field-data) offers distinct value for optimizing your user experience.
+## Collect your own field data if possible
+
+While lab tools are invaluable, [the data they provide isn't always predictive of how a website performs for real users](https://discuss.httparchive.org/t/lighthouse-scores-as-predictors-of-page-level-crux-data/2232).
+
+For example, Lighthouse runs tests with simulated throttling in a simulated desktop or mobile environment. While such simulations of slower network and device conditions often help surface user experience problems better than native network and device conditions, [they're just a single slice](/lab-and-field-data-differences/) of the large variety in network conditions and device capabilities across a website's entire user base.
+
+{% Aside %}
+To learn more about how simulated throttling works and how it differs from other types of throttling, [this bit of Lighthouse documentation](https://github.com/GoogleChrome/lighthouse/blob/master/docs/throttling.md#network-throttling) is your time.
+{% endAside %}
+
+### First or third-party tools
+
+The best dataset for assessing website performance in the field is the one _you_ build. That starts with collecting field data from your website's visitors. How you do this depends on the size of your organization, and whether you want to pay for a third-party solution or create your own.
```

Comment:
We don't typically promote third-party solutions on web.dev, but in case someone has no idea what RUM tools are available, it might be worth linking to https://en.wikipedia.org/wiki/Real_user_monitoring#RUM_software, WDYT?